### ISSUE TITLE
Fix list design & add in font

### ISSUE THREAD

**Issue by fvcproductions** [fvcproductions/apprenticeships.me#40]
There is a one-off in card row design - potentially caused by Algolia paginate configuration.

![screen shot 2019-01-13 at 8 38 35 pm](https://user-images.githubusercontent.com/4284691/51096697-54a09a00-1773-11e9-8d26-abedbf4c5e8d.jpg)

Source Sans Pro is not loaded through Google Fonts - need to import that in or choose a new font.

https://fonts.google.com/specimen/Source+Sans+Pro?selection.family=Source+Sans+Pro

`https://fonts.google.com/specimen/Source+Sans+Pro?selection.family=Source+Sans+Pro`

---

> ✅️ By submitting this issue, I have verified the following

- [x] Checked to see if the issue has already been discussed before 🤔️
- [x] If proposing new content to be added, made sure enough details were provided 🔍️

**Comment by fvcproductions** [comment]
@benjaminmodayil 

To add on to the list design part, I'm not that familiar with Algolia but I think it would be best to show a lot more results on each page - I'm thinking like 15 at least because that's the main point of the website - to showcase the apprenticeships. 

Once we have images for each apprenticeship, I think we could make the paginated list size a lot smaller - to say 9 or 12.

**Comment by benjaminmodayil** [comment]
Yup. I remember seeing this option before in the Algolia settings. It should help with the offset. Except when it shrinks to a two column though. I guess one version will always have an offset. I'll switch it to 15 for now and update you here when it's done.

**Comment by benjaminmodayil** [comment]
It's updated to 15 😄

Let me know if you want me to change it.

**Comment by benjaminmodayil** [comment]
Also, if you rather have the Algolia account under your email, let me know. I think that probably would be best for the longevity of the project since you are the repository owner.

**Comment by fvcproductions** [comment]
@benjaminmodayil Do they allow multiple people to own a project for free like with Netlify? If so, feel free to add me in using `fvcproductions@gmail.com`. 

Otherwise, I'm okay with you transferring as well.

**Comment by fvcproductions** [comment]
@benjaminmodayil Thanks for updating!

**Comment by fvcproductions** [comment]
@benjaminmodayil Also feel free to close this issue since both bugs were addressed 👍

**Comment by benjaminmodayil** [comment]
@fvcproductions Seems that there isn't a free option that allows multiple people to have access. I'll talk to you about transferring the account after working on #37
